prometheus/v2.22.1/Dockerfile

```diff
@@ -0,0 +1,42 @@
+## Use original image to copy files from
+## ref: https://github.com/prometheus/prometheus/blob/00f16d1ac3a4c94561e5133b821d8e4d9ef78ec2/Dockerfile
+FROM quay.io/prometheus/prometheus:v2.22.1 as builder
+
+## Build RedHat compliant image
+ARG ARCH="amd64"
+ARG OS="linux"
+FROM registry.access.redhat.com/ubi8/ubi:8.4
+LABEL maintainer="The Prometheus Authors <prometheus-developers@googlegroups.com>"
```

Comment:
I think that we will maintain UBI based image I suggest setting maintainers to "collection@sumologic.com"